packages/jest-runner/fuzz.ts

```diff
@@ -14,23 +14,26 @@
  * limitations under the License.
  */
 
+import * as fs from "fs";
+
 import { Circus, Global } from "@jest/types";
+
+import {
+	defaultOptions,
+	Options,
+	startFuzzingNoInit,
+	asFindingAwareFuzzFn,
+	FindingAwareFuzzTarget,
+} from "@jazzer.js/core";
```

Comment:
Can we maybe auto-order these names, too? :)

I looked a bit around, and it seems like [sort-imports](https://eslint.org/docs/latest/rules/sort-imports) should be able to do this, but we would probably need to set `"ignoreDeclarationSort": false ` ([link](https://eslint.org/docs/latest/rules/sort-imports#ignoredeclarationsort)) in order for it not to clash with the configuration in this PR.

Reply:
Good point! Added `sort-import` as suggested. Much better now :smile: